frontend/src/features/workspace/components/WorkspaceFormRow/WorkspaceFormRow.tsx

```diff
@@ -23,6 +28,12 @@ const RELATIVE_DATE_FORMAT = {
   sameElse: 'D MMM YYYY h:mma', // 6 Oct 2021 9:30pm
 }
 
+const RESPONSE_MODE_TO_TEXT: { [key in FormResponseMode]: string } = {
+  [FormResponseMode.Multirespondent]: 'Multi-respondent form',
+  [FormResponseMode.Email]: 'Email form',
+  [FormResponseMode.Encrypt]: 'Storage form',
+}
+
```

Comment:
@KenLSM  Could we export this and use it in `GeneralTabHeader.tsx` as well?

Reply:
Good catch! Forgot about that. 42af61104